package.json

```diff
@@ -8,10 +8,14 @@
   "description": "API for SPARK",
   "scripts": {
     "migrate": "node bin/migrate.js",
-    "start": "node bin/spark.js",
+    "start:api": "node spark-api/bin/spark.js",
+    "start:publish": "node spark-publish/bin/spark-publish.js",
+    "test:api": "mocha spark-api/test/**/*.js",
+    "test:publish": "mocha spark-publish/test/**/*.js",
     "test": "standard && mocha"
```

Comment:
should this instead do

```bash
npm-run-all test:*
```

Reply:
> should this instead do
> 
> ```shell
> npm-run-all test:*
> ```

It's an option too. Let's discuss the difference:

- In my version, we run a single Mocha process that can run tests for all components in parallel.
- In your version, we run multiple Mocha processes, one for each component. I checked the docs and nom-run-all supports both parallel and serial execution, which is great. 

  The downside is that we will call `npm run` several times, which adds overhead and slows down the feedback loop. I did a quick test, and it seems that `npm run` adds 160ms.

```
❯ time (echo ok)                                                                                   
ok
( echo ok; )  0.00s user 0.00s system 39% cpu 0.002 total
❯ time npm run echo

> @filecoin-station/spark-api@0.0.0 echo
> echo ok

ok
npm run echo  0.16s user 0.03s system 105% cpu 0.186 total
```

Reply:
Oh I see, I didn't realize that `mocha` would run all test suites. That's perfect then!